### PR TITLE
[Julia/en] Julia0.5

### PR DESCRIPTION
**Pull request by edrlima**
Julia 0.5 update

- Change ASCIIString -> String
- Change Error Messages
- Change name function f -> g // f variable already defined

### CODE REVIEW

**Comment by edrlima** [comment]
Sorry ... it's just the second commit (Julia 0.5 update).. the first has a temporary file that I created

File: julia.html.markdown [hunk 1]
```diff
@@ -192,8 +192,8 @@ sort!(arr) # => [4,5,6]; arr is now [4,5,6]
 
 # Looking out of bounds is a BoundsError
 try
-    a[0] # => ERROR: BoundsError() in getindex at array.jl:270
-    a[end+1] # => ERROR: BoundsError() in getindex at array.jl:270
+    a[0] # => BoundsError([7,2,4,3,4,5,6],(0,))
+    a[end+1]
```
Comment (vendethiel) [inline julia.html.markdown]: No error for this one?

**Comment by edrlima** [comment]
In Juno the error appear only for the first error...probably after the first exception happens the code goes directly to the catch block

File: julia.html.markdown (hunk 1 above)
Comment (edrlima) [inline julia.html.markdown]: In Juno the error appears only for the first error...probably after the first exception happens the code goes directly to the catch block

**Comment by vendethiel** [comment]
Going to close this one for now :). Feel free to ping us here if you find some time working on it again! Thanks.

**Pull request closed without merging**